kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/SectionEditor.vue

```diff
@@ -16,6 +194,183 @@
+        message: 'Section title',
+        context: 'Label for section settings title input field.',
+      },
+      numberOfQuestion: {
+        message: 'Number of questions',
+        context: 'label for section settings number of questions input field.',
+      },
+      descriptionLabel: {
+        message: 'Description (optional)',
+        context: 'Label for the section settings description input field.',
+      },
+      questionOrder: {
+        message: 'Question order',
+        context: 'Heading for the question order',
+      },
+      randomizedLabel: {
+        message: 'Randomized',
+        context: 'Radio button label for displaying random questions to learners.',
+      },
+      randomizedOptionDescription: {
+        message: 'Each learner sees a different question order',
+        context: 'Describes how the randomize radio button works when selected.',
+      },
+      fixedLabel: {
+        message: 'Fixed',
+        context: 'Radio button label for displaying same question order to learners.',
+      },
+      fixedOptionDescription: {
+        message: 'Each learner sees the same question order',
+        context: 'Describes how the fixed radio button works when selected.',
+      },
+      quizResourceSelection: {
+        message: 'Quiz resource selection',
+        context: 'Subtitle for quiz resource selection in the section settings side panel.',
+      },
+      selectedResources: {
+        message: '{count} resources selected from  {count} channels',
+        context: "Only translate  'resources selected from' and 'channels'.",
+      },
+      changeResourceButton: {
+        message: 'Change resources',
+        context: 'Button for changing the resources in the section settings sidepanel.',
+      },
+      sectionOrderTitle: {
+        message: 'Section order',
+        context: 'Subheading for section ordering in the section settings sidepanel.',
+      },
+      currentSection: {
+        message: 'Current Section',
+        context: 'label for the currently ordered section.',
+      },
+      deleteSectionButton: {
+        message: 'Delete section',
+        context: 'Button for deleting section.',
+      },
+      applySettingsButton: {
+        message: 'Apply settings',
+        context: 'Button to save the new section settings.',
+      },
+      uniqueTitle: {
+        message: 'Section 2 / unique title',
+        context: 'Label for the other sections other than the current section.',
+      },
+    },
   };
 
 </script>
+
+
+<style lang="scss" scoped>
+
+  @import '~kolibri-design-system/lib/styles/definitions';
+
+  .number-field {
+    display: inline-block;
+    max-width: 31em;
+    margin-right: 0.5em;
+  }
+
+  .section-settings-heading {
+    font-size: 1em;
+    font-weight: 600;
+  }
+
+  .section-settings-top-heading {
+    font-size: 18px;
+    font-weight: 600;
+  }
+
+  .section-order-list {
+    height: 2.5em;
+    margin-top: 0.5em;
+    border: 1px solid #dedede;
```

Comment:
For all of the colors throughout the css here, every time you are using a hex code, please swap this out with one of our theme tokens. To do this, it will mean [adding it in the template](https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/views/AppBar.vue#L71), or [calculating in computed](https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/coach/assets/src/views/common/Score.vue#L44) as needed, rather than setting it here in the css. But, it means that our theming will not break :) There are several places throughout this PR where this is used, and this update should be applied to all of them.